src/core/controllers/oidc.go

```diff
@@ -82,6 +82,15 @@ func (oc *OIDCController) Callback() {
 		oc.SendBadRequestError(errors.New("State mismatch"))
 		return
 	}
+
+	error := oc.Ctx.Request.URL.Query().Get("error")
```

Comment:
Could you rename the var to err?

Reply:
Hi @reasonerjt, no I can't, it is not an go 'error' type but an error_code string. I'll change the name to errorCode to make it clear :)